.travis.yml

```diff
@@ -4,11 +4,16 @@ python:
 
 install:
   - pip install pylama
+  - pip install tox
```

Comment:
You can do that in a single line. Just separate `tox` with a space from `pylama`